.github/ISSUE_TEMPLATE/rebase.md

```diff
@@ -1,5 +1,13 @@
 # Rebase to a new version of Fedora (N)
 
+## At previous Fedora major release
+
+### Open tickets to track related work for this release
+
+- [ ] Fedora Changes Considerations [example](https://github.com/coreos/fedora-coreos-tracker/issues/918)
```

Comment:
Insignificant nit: maybe parenthesize `example`?

Reply:
updated.. Also updated the example links to the most recent ones.